README.md

```diff
@@ -48,8 +48,6 @@ First install Vagrant following [the steps in the official documentation](http:/
 
 Finally, follow the instructions displayed by the virtual machine.
 
-**Note:** Changes to `app.rb` will require you to restart the webserver process.
```

Comment:
Rather than simply removing this, I think it might be better to state that new reality explicitly, especially as the new behaviour doesn't seem to be obvious, and people's won't necessarily expect that this will happen.

Reply:
Good call @tmtmtmtm :+1:

I've added this back in 15a615b and moved this and the note about Sass to a more appropriate section in 7c4ced4 (tidying as I go).